openfl/interface/aggregator.py

```diff
@@ -36,7 +36,13 @@ def aggregator(context):
         default='plan/cols.yaml', type=ClickPath(exists=True))
 @option('-s', '--secure', required=False,
         help='Enable Intel SGX Enclave', is_flag=True, default=False)
-def start_(plan, authorized_cols, secure):
+@option('-c', '--cert_path',
+        help='The cert path where pki certs will reside', required=False)
+@option('--fqdn', required=False, type=click_types.FQDN,
+        help=f'The fully qualified domain name of'
+             f' aggregator node [{getfqdn_env()}]',
+        default=getfqdn_env())
```

Comment:
Aggregator FQDN could be read from the plan. According to the CLI flow, the plan should be initialized by the step of aggregator start.

Reply:
I agree with Iliya, this option overrides information in plan.yaml.
Why would we need this?